requirements.txt

```diff
@@ -0,0 +1,4 @@
+﻿numpy
+open3d>=0.15.2
```

Comment:
I would stick with `==` and not `>=` for now, open3d has been changing in interesting and API breaking ways! It's really easy to have multiple python version using virtualenv or conda env

Reply:
Updated